plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/thrift/ThriftHiveMetastoreClient.java

```diff
@@ -433,7 +463,10 @@ public List<Role> listRoles(String principalName, PrincipalType principalType)
     public List<HiveObjectPrivilege> listPrivileges(String principalName, PrincipalType principalType, HiveObjectRef hiveObjectRef)
             throws TException
     {
-        return client.listPrivileges(principalName, principalType, hiveObjectRef);
+        return client.listPrivileges(principalName, principalType,
```

Comment:
The convention is to put either all the parameters on the same line or on separate lines.
Maybe just create the new hiveObjectRef before the method call